apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/ExecuteHelper.cs

```diff
@@ -55,7 +62,10 @@ internal static bool CheckForSessionExpiredError(this RpcException rpcException,
             return false;
         }
 
-        internal static bool IsSessionExpiredError(this RpcException rpcException)
+        /// <summary>
+        /// Determines whether <paramref name="rpcException"/> is due to a session expiry.
+        /// </summary>
+        public static bool IsSessionExpiredError(this RpcException rpcException)
```

Comment:
unfortunately, yes and its where I got it.

https://github.com/GoogleCloudPlatform/google-cloud-java/blob/df975b0354273cc984c8cda2f31d2697b770b3b3/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java#L689